scheduler/cups-driverd.cxx

```diff
@@ -1555,6 +1555,24 @@ list_ppds(int        request_id,	/* I - Request ID */
     }
   }
 
+ /*
+  * Free include, exclude, matches, requested, no longer needed
+  */
+  if (include != NULL)
+    cupsArrayDelete(include);
+  if (exclude != NULL)
+    cupsArrayDelete(exclude);
+  if (matches != NULL && matches != PPDsByMakeModel) 
+    cupsArrayDelete(matches);
+  if (requested != NULL)
+    cupsArrayDelete(requested);
+
+  /*
+   * Free device_id_re and make_and_model_re, no longer needed
+   */
+  free(device_id_re);
+  free(make_and_model_re);
+
```

Comment:
We don't care about this because main() calls list_ppds() and then returns, freeing all memory used by the program. Perhaps I should have it not return and instead exit().